chainstate/src/interface/chainstate_interface.rs

```diff
@@ -131,6 +132,11 @@ pub trait ChainstateInterface: Send {
         &self,
         block_index: &BlockIndex,
     ) -> Result<Option<BlockReward>, ChainstateError>;
+
+    // Returns epoch data for the given epoch index.
```

Comment:
Three slashes for documentation of functions

Reply:
Fixed too.